projects/storefrontlib/src/cms-components/misc/icon/icon-loader.service.ts

```diff
@@ -34,6 +35,14 @@ export class IconLoaderService {
     }
   }
 
+  /**
+   * Indicates the icon direction (ltr vs rtl). The icon direction is configurable, but
+   * optional, as only a few icons should be flipped for rtl direction.
+   */
+  getFlipDirection(type: ICON_TYPE | string): DirectionMode {
+    return this.config.flipDirection[type];
```

Comment:
Edge case: `flipDirection` can be undefined (when customer dismantles deeply our modules).
```suggestion
    return this.config?.flipDirection?.[type];
```